src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1513,7 +1530,10 @@ def setupSite(self, scriptCfg, cfg=None):
             gLogger.notice("Registering System instances")
             for system in setupSystems:
                 self.addSystemInstance(system, self.instance, self.setup, True)
+
             for system, service in setupServices:
+                addTornado = bool(service.startswith("Tornado"))
```

Comment:
```suggestion
                addTornado = service.startswith("Tornado")
```
The `bool` is superfluous, is it not?